addons/mrp/models/mrp_workorder.py

```diff
@@ -409,6 +408,12 @@ def record_production(self):
             self.button_finish()
         return True
 
+    @api.multi
+    def _start_nextworkorder(self):
+        for record in self:
```

Comment:
one write for better performance : `self.mapped('next_work_order_id').filtered(lambda x: x.state == 'pending').write({'state':'ready'})`